python-frontend/src/main/java/org/sonar/python/semantic/ProjectLevelSymbolTable.java

```diff
@@ -226,8 +226,8 @@ public void visitCallExpression(CallExpression callExpression) {
       }
     }
 
-    private static boolean isCallRegisteringDjangoView(CallExpression callExpression) {
-      TypeChecker typeChecker = new TypeChecker(new BasicTypeTable());
+    private boolean isCallRegisteringDjangoView(CallExpression callExpression) {
+      TypeChecker typeChecker = new TypeChecker(new BasicTypeTable(new ProjectLevelTypeTable(ProjectLevelSymbolTable.this)));
```

Comment:
_nit_ This `TypeChecker` construction is quite unclear to me (3 constructors on the same line, `ProjectLevelTypeTable` and `ProjectLevelSymbolTable` have so similar naming) but I guess it is another battle :wink: Ok like this!